src/2_storage.js

```diff
@@ -162,13 +162,13 @@ var cookies = function(perm) {
 			return returnCookieObject;
 		},
 		get: function(key) {
```

Comment:
Can we add a unit test on this, seems like a relatively simple unit test should build our confidence here and prevent regressions.

Reply:
👍  I've added unit tests for all of the cookie storage methods and have fixed a few bugs in the process.